tools/cockpit.spec

```diff
@@ -224,6 +233,10 @@ BuildRequires:  python3-tox-current-env
 %patch101 -p1
 %patch102 -p1
 %endif
+# For anything based on SLES 15 codebase (including Leap, SLEM)
+%if 0%{?sle_version} >= 150400 && 0%{?sle_version} <= 150700
```

Comment:
If we just have `%if 0%{?suse_version} == 1500`, then it should be sufficient. We don't care about out of support (or soon to be) Leaps and SLES and if they don't build.

Reply:
Yes, that makes sense. I wanted to avoid < than 15 SP4, as in older streams we still don't have anything newer than python3.6 (aside from minimal 3.10 etc)  I believe that was also one of blockers for newer cockpits.